gadfly/gap_fill_generator.py

```diff
@@ -30,15 +30,28 @@ def find_named_entities(self):
         return entities
 
     def summarize_sentences(self):
-        fs = FrequencySummarizer()
-        sents = []
-        for span in self._parsed_text.sents:
-            sent = [self._parsed_text[i] for i in range(span.start, span.end)]
-            tokens = []
-            for token in sent:
-                tokens.append(token.text)
-            sents.append(tokens)
-        sentences = fs.summarize(sents, 5)
+        option = 1
```

Comment:
If it is a switch between the two different types of summarizers, we would be better off calling the summarizer explicitly. Do you mean when running from terminal as an arg? If so, that makes sense  and I'll implement. Though, I imagined that having different summarizers may or may not be a long-term thing and don't know the impact elsewhere from changing how the program is run.